Description: Patch is applied to fix several build issues.
 Remove compiler optimizations.
Author: Jaromír Mikeš <mira.mikes@seznam.cz>
Author: Dennis Braun <d_braun@kabelmail.de>
Last-Update: 2020-02-02
Forwarded: no

--- a/source/Makefile
+++ b/source/Makefile
@@ -33,8 +33,7 @@
 
 CPPFLAGS += -I. -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS
 CPPFLAGS += -DENABLE_VECTOR_MODE 
-CXXFLAGS += -fPIC -Wall -ffast-math -funroll-loops -O2
-CXXFLAGS += -march=native
+CXXFLAGS += -fPIC -Wall -ffast-math -funroll-loops
 LDLFAGS += 
 LDLIBS +=
 
@@ -48,7 +47,7 @@
 
 
 $(ZITA-CONVOLVER_MIN):	$(ZITA-CONVOLVER_O)
-	$(CXX) -shared $(LDFLAGS) -Wl,-soname,$(ZITA-CONVOLVER_MAJ) -o $(ZITA-CONVOLVER_MIN) $(ZITA-CONVOLVER_O) $(ZITA-CONVOLVER_DEP)
+	$(CXX) $(LDFLAGS) -shared -Wl,-soname,$(ZITA-CONVOLVER_MAJ) -o $(ZITA-CONVOLVER_MIN) $(ZITA-CONVOLVER_O) $(ZITA-CONVOLVER_DEP)
 
 
 install:	$(ZITA-CONVOLVER_MIN)
@@ -56,8 +55,8 @@
 	install -d $(DESTDIR)$(LIBDIR)
 	install -m 644 $(ZITA-CONVOLVER_H) $(DESTDIR)$(INCDIR)
 	install -m 755 $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(LIBDIR)
-	ldconfig
-	ln -sf $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(LIBDIR)/$(ZITA-CONVOLVER_SO)
+	ln -sf $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(LIBDIR)/$(ZITA-CONVOLVER_MAJ)
+	ln -sf $(ZITA-CONVOLVER_MAJ) $(DESTDIR)$(LIBDIR)/$(ZITA-CONVOLVER_SO)
 
 uninstall:
 	rm -rf $(DESTDIR)$(INCDIR)/$(ZITA-CONVOLVER_H)
